wandb/sdk/internal/internal_api.py

```diff
@@ -171,7 +171,7 @@ class Api:
         Override the settings here.
     """
 
-    HTTP_TIMEOUT = env.get_http_timeout(10)
+    HTTP_TIMEOUT = env.get_http_timeout(60)
```

Comment:
probably too long

Reply:
How about 30s? 60s matches the server-side timeout.

Reply:
Yea, let's do 30s